main.go

```diff
@@ -242,6 +243,15 @@ func main() {
 	clustersCtrl := controller.NewClustersController(service, config)
 	app.MountClustersController(service, clustersCtrl)
 
+	// Mount "resources" controller
+	resourcesCtrl := controller.NewResourceController(service, appDB)
+	app.MountResourceController(service, resourcesCtrl)
+
+	// Mount "organizations" controller
+	organizationService := model.NewOrganizationModelService(db, appDB)
+	organizationsCtrl := controller.NewOrganizationController(service, appDB, organizationService)
```

Comment:
Slight change here
```
organizationModelService := model.NewOrganizationModelService(...)
organizationService := model.NewOrganizationService(organizationModelService)
organizationCtrl := model.NewOrganizationController(service,appDB,organizationService)
```

Isn't this what we discussed :) ? had kinda outlined so in https://gist.github.com/sbose78/4bb7d589014ee896958956981c84f1a7
@alexeykazakov @sbryzak